src/main/java/uk/gov/hmcts/reform/prl/enums/manageorders/CaseTransferOptionsEnum.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.prl.enums.manageorders;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import lombok.RequiredArgsConstructor;
+import uk.gov.hmcts.reform.prl.enums.CustomEnumSerializer;
+
+@RequiredArgsConstructor
+@JsonSerialize(using = CustomEnumSerializer.class)
+public enum CaseTransferOptionsEnum {
```

Comment:
If this enum is related to C49 order, then this might need to be removed

Reply:
@cshiwarale as discussed we disabled this option from UI.